modules/olmv1-updating-an-operator.adoc

```diff
@@ -159,6 +161,53 @@ spec:
 ----
 <1> Update the version from `1.11.1` to `1.12.1`
 
+** If you want to define a range of acceptable update versions, edit your CR similar to the following example:
++
+.Example CR with a version range specified
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: pipelines-operator
+spec:
+  packageName: openshift-pipelines-operator-rh
+  version: >1.11.1, <1.13 <1>
+----
+<1> Specifies that the desired version range is greater than version `1.11.1` and less than `1.13`. For more information, see "Support for version ranges".
+
+** If you want to update to the latest version that can be resolved from a channel, edit your CR similar to the following example:
```

Comment:
I think this bit is unclear, there's no guarantee that all operators will have a `latest` channel. I would maybe call out that this operator happens to have a channel called `latest` or maybe select a different channel example?

Reply:
Good point. I didn't think of that. I will pick a different channel from the pipelines Operator

Reply:
Thanks @kevinrizza! I addressed both of your comments in https://github.com/openshift/openshift-docs/pull/71400/commits/1a454da70484267a60c68b64dde7d8fcca0d2c08.